scenarioo-client/test/spec/shared/navigation/branchOrderBy.filter.spec.js

```diff
@@ -170,8 +233,57 @@ describe('Filter scBranchOrderBy', function () {
             expect(result[2].branch.name).toEqual('Ba');
             expect(result[3].branch.name).toEqual('Ae');
         });
+    });
+
+    describe('should order given branch resource objects by name-ascending:', function () {
+
+        it('initConfig', function () {
+            initConfig(CFG_NAME_ASCENDING);
+        });
+
+        it('then alphabetically (not case sensitive!)', function () {
+
+            var result = scBranchOrderByFilter(DEFAULT_INPUT);
+
+            expect(result[0].branch.name).toEqual('Ba');
+            expect(result[1].branch.name).toEqual('be');
+            expect(result[2].branch.name).toEqual('aa');
+            expect(result[3].branch.name).toEqual('Ae');
+        });
+    });
+
+    describe('should order given branch resource objects by config not correct. Default: name-ascending:', function () {
```

Comment:
`Should order given branch resource objects by name ascending as a default` - "config not correct" is missleading, since the passed config does not contain an invalid value.